scan/report-es.tex

```diff
@@ -0,0 +1,273 @@
+ el contacto con otras personas. Si tiene otros problemas de salud o está muy
+ enfermo, o si siente preocupación por su enfermedad, debe comunicarse con su
+ proveedor de atención médica. Si los síntomas persisten, debe hablar sobre la
+ posibilidad de realizarse otro análisis para detectar la COVID-19 con su
+ proveedor de atención médica.
+
+
+\item
+
+  \textbf{El centro en el que se realiza el análisis informará los casos en los
+  que los análisis arrojen resultados positivos (en los que se indique
+  ``DETECTADO'' o ``Virus detectado'') al departamento de Salud del condado o
+  estado correspondiente.} NO es necesario que informe estos resultados usted
+  mismo. Si el resultado de su análisis es positivo para el nuevo coronavirus,
+  cabe la posibilidad de que Salud Pública realice un seguimiento de su caso, y
+  le haga preguntas sobre su enfermedad y las personas con las que pudo haber
+  estado en contacto.
+
+\item
+
+  \textbf{Puede imprimir los resultados de su análisis o enviárselos por correo
+  electrónico a su proveedor de atención médica.}
+
+\end{itemize}
+
+\bigskip
+\textbf{Metodología de la PCR cualitativa para el SARS-CoV-2 (COVID-19):}
+
+Este análisis está diseñado para detectar el SARS-CoV-2 por medio de la PCR con
+transcripción inversa. El análisis se basa en la amplificación de los segmentos
+Orf1B y S del genoma viral con control interno sobre la amplificación de la
+ribonucleasa P (RNasa P) humana. El Northwest Genomics Center (NWGC) de la
+University of Washington desarrolló este análisis y determinó sus
+características de funcionamiento. El análisis desarrollado por este laboratorio
+está validado y fue presentado ante la Food and Drug Administration de los EE. 
+UU. para obtener la autorización de uso de emergencia. La evaluación
+independiente de la validación del análisis por parte de la FDA se encuentra
+pendiente. Este laboratorio cuenta con la certificación de las Clinical
+Laboratory Improvement Amendments (CLIA, enmiendas para la mejora de
+laboratorios clínicos) y está calificado para realizar análisis de laboratorio
+clínicos de alta complejidad.
+
+Para obtener más información acerca del proveedor, consulte los siguientes
+enlaces proporcionados por el Washington State Department of Health:
+
+\begin{itemize}
+\item
+
+  \link{https://www.doh.wa.gov/Portals/1/Documents/1600/coronavirus/COVIDcasepositive.pdf}{Patients
+  with confirmed or suspected COVID-19 (Pacientes con sospecha o confirmación de
+  la COVID-19)}
+
+\item
+
+  \link{https://www.doh.wa.gov/Portals/1/Documents/1600/coronavirus/COVIDexposed.pdf}{Patients
+  who were exposed to a confirmed COVID-19 case (Pacientes que estuvieron
+  expuestos a un caso confirmado de la COVID-19)}
+
+\item
+
+  \link{https://www.doh.wa.gov/Portals/1/Documents/1600/coronavirus/COVIDconcerned.pdf}{Unexposed
+  patients with COVID-19 symptoms (Pacientes que no estuvieron expuestos y que
+  tienen síntomas de la COVID-19)}
```

Comment:
WA DOH provides these documents in many other languages than English, so I think these links should point to the appropriate language, when available. For example, the Spanish version of the first link is https://www.doh.wa.gov/Portals/1/Documents/1600/coronavirus/COVIDcasepositiveSPANISH.pdf.

I don't know if that URL pattern is consistent, but I believe the matrix of documents × languages is listed on this page: https://www.doh.wa.gov/Emergencies/NovelCoronavirusOutbreak2020/HealthEducation

Reply:
I updated the translated documents for Spanish and Vietnamese. No translations were available for Chinese.

@tsibley Oddly enough, the latest commit that changes the URL of these links breaks the build. Do you have any idea why?

Reply:
> No translations were available for Chinese.

Wow, wasn't expecting that.

> Oddly enough, the latest commit that changes the URL of these links breaks the build. Do you have any idea why?

In LaTeX, underscores are special chars, so the ones in the new URLs need to be escaped like `\_`.

Reply:
Tricky LaTeX! It's now fixed. Thanks for the help.